### PR TITLE
Orders Api Integration

### PR DESCRIPTION
**Pull request by riagarg5**
Order Api integration and signature verification in test app

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,6 +7,17 @@
   <artifactId>razorpay-java-testapp</artifactId>
   <version>1.0-SNAPSHOT</version>
 
+
+  <repositories>
```
Comment (ankit-rzp) [inline pom.xml]: remove this

File: pom.xml [hunk 2]
```diff
@@ -30,7 +41,7 @@
     <dependency>
       <groupId>com.razorpay</groupId>
       <artifactId>razorpay-java</artifactId>
-      <version>1.0.0</version>
+      <version>1.2.0-SNAPSHOT</version>
```
Comment (ankit-rzp) [inline pom.xml]: if you have tested `1.2.0` then first release that, and use `1.2.0`, not the snapshot version

File: src/main/java/com/razorpay/PaymentResource.java [hunk 3]
```diff
@@ -27,8 +27,14 @@ public PaymentResource(RazorpayClient client) {
 
   @GET
   @Produces(MediaType.TEXT_HTML)
-  public View getPaymentForm() {
-    return new PaymentView(amount);
+  public View getPaymentForm() throws RazorpayException {
+    JSONObject options = new JSONObject();
+    options.put("amount", amount); // Note: The amount should be in paise.
+    options.put("currency", "INR");
+    options.put("receipt", "txn_123456");
```
Comment (ankit-rzp) [inline src/main/java/com/razorpay/PaymentResource.java:34]: i dont think `receipt` is mandatory, check doc once.

File: src/main/java/com/razorpay/PaymentResource.java [hunk 4]
```diff
@@ -37,13 +43,21 @@ public View getPaymentForm() {
...
+        && StringUtils.isNotBlank(orderId)) {
       try {
-        Payment payment = client.Payments.capture(paymentId, options);
-        return Response.ok(payment.toString()).build();
+        options.put("razorpay_payment_id", paymentId);
+        options.put("razorpay_order_id", orderId);
+        options.put("razorpay_signature", razorpaySignature);
+        boolean isEqual = client.Utility.verifyPaymentSignature(options);
+
+        if (isEqual) {
```
Comment (ankit-rzp) [inline src/main/java/com/razorpay/PaymentResource.java:58]: if not equal then throw exception

**Review by ankit-rzp — CHANGES_REQUESTED** [review]
Left comments. Also update the description, and use the correct formatter.

**Review by mb-14 — COMMENTED** [review]
Make the changes @ankit-rzp has requested. Otherwise LGTM.

File: src/main/java/com/razorpay/PaymentResource.java (hunk 3 above)
Comment (riagarg5) [reply src/main/java/com/razorpay/PaymentResource.java:34]: It is required

File: src/main/java/com/razorpay/PaymentResource.java (hunk 4 above)
Comment (riagarg5) [reply src/main/java/com/razorpay/PaymentResource.java:58]: If not equal we are returning bad request

**Review by ankit-rzp — APPROVED** [review]
LGTM: But before you merge, please make sure version `1.2.0` is updated in mvn central

**Pull request merged**